guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -1259,7 +1259,6 @@ WEBVTT
 
 54:24.455 --> 54:35.483
 [J Jay Couey]: So, instead of speaking for an hour, I already probably spoke too long, I want to be able to answer as many questions as anybody wants to throw at me, even from previous talks.
-
 [Charles Kovess]: Wonderful.
```

Comment:
This is what that change looks like. 
![image](https://github.com/user-attachments/assets/bc9cfa4e-956c-4533-87fc-ae299bf34c31)